commcare_connect/opportunity/tests/test_views.py

```diff
@@ -512,3 +513,127 @@ def test_not_found_invite_with_found_user(self, mock_fetch_users, mock_update_an
         assert response.status_code == 200
         assert response.headers["HX-Redirect"] == self.expected_redirect
         mock_update_and_send.assert_called_once_with(mock_user, self.opportunity.id)
+
+
+def test_views_use_opportunity_decorator_or_mixin():
```

Comment:
Would this fail for a view that inherits from a view with the proper authentication? Like a base view class of sorts?